src/components/main/flight/CarbonEmissionsScreen.js

```diff
@@ -66,11 +66,36 @@ export default class CarbonEmissionsScreen extends React.Component {
 
   //Calculate emissions using distance and seat class
   calcEmissions() {
-    let temp = this.props.navigation.getParam('distance', 'distanceTraveled');
-    tempD = Math.round(temp);
-    tempF = tempD / 1000;
+    let dist = this.props.navigation.getParam('distance', 'distanceTraveled');
```

Comment:
All distance /class multipliers should be definied as constants on a separate file in utilities for easy future changes (i.e. new calculations or new airplanes)